src/assets/mixins.styl

```diff
@@ -0,0 +1,16 @@
+//fonts mixins
+play($weight)
+  font-family 'Playfair Display', serif
```

Comment:
consider making this font a variable for the whole project, if you don't think that's necessary please respond why in this thread.